src/main/java/org/fundacion/jala/converter/view/controllers/ClientRequest.java

```diff
@@ -46,44 +45,61 @@
     private final String defaultCharset = "UTF-8";
     private static final Logger LOGGER = LogManager.getLogger();
 
-
     /**
-     * Creates a request given a requestForm.
+     * Http client creates a request given a requestForm.
      */
     public ClientRequest() {
         this.httpClient = HttpClients.createDefault();
     }
 
     /**
      * Executes a request given the type of requestForm.
-     *
      * @return sResponse
      * @throws ClientProtocolException
      * @throws IOException
      */
-    public String executeRequest(final IRequestForm newRequestForm) throws ClientProtocolException, IOException {
-        this.requestForm = newRequestForm;
+    public String executeRequest(final IRequestForm newRequestForm, final String token) throws ClientProtocolException, IOException {
+        requestForm = newRequestForm;
         httpPost = new HttpPost(requestForm.getURL());
         builder = MultipartEntityBuilder.create();
         addBodyFields();
         multipart = builder.build();
         httpPost.setEntity(multipart);
-        httpPost.setHeader("Authorization", "Bearer " + findUserById(Integer.parseInt(dotenv.get("USER_ID"))).getToken());
+        httpPost.setHeader("Authorization", "Bearer " + token);
         CloseableHttpResponse response = httpClient.execute(httpPost);
         HttpEntity responseEntity = response.getEntity();
         String sResponse = EntityUtils.toString(responseEntity, defaultCharset);
         return sResponse;
     }
 
+    /**
+     * Executes a request given the type of requestForm
+     * @param newRequestForm the request form
+     * @return a String with the response
+     * @throws ClientProtocolException when an error on the HTTP protocol occurs
+     * @throws IOException when an invalid input is provided
+     */
+    public String executeRequestWithoutToken(final IRequestForm newRequestForm)
+            throws ClientProtocolException, IOException {
+        requestForm = newRequestForm;
+        httpPost = new HttpPost(requestForm.getURL());
+        builder = MultipartEntityBuilder.create();
+        addBodyFields();
+        multipart = builder.build();
+        httpPost.setEntity(multipart);
+        CloseableHttpResponse response = httpClient.execute(httpPost);
+        HttpEntity responseEntity = response.getEntity();
+        String sResponse = EntityUtils.toString(responseEntity, "UTF-8");
+        return sResponse;
+    }
+
     /**
      * Downloads a file from endpoint to given path
      * @throws ClientProtocolException
      * @throws IOException
      */
     public void download(final String filePath) throws IOException {
-        String localToken = authGetToken();
         String sURL = "http://localhost:8080/api/download/";

```

Comment:
URL could be set as a final attribute or be declared in a properties file.

Reply:
changed it.